front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -165,6 +166,30 @@ export default function AssistantBuilder({
       openedAt: template != null ? Date.now() : null,
     });
 
+  // We deactivate the Preview button if the BuilderState is empty (= no instructions, no tools)
+  // Needs the regex cause tiptap always keeps a node, which is a \n when empty.
+  const isBuilderStateEmpty =
```

Comment:
can be simplified as:
```
!builderState.instructions?.trim() && !builderState.actions.length
```

because trim removes all leading / trailing ws including newlines + empty string are falsy in js

Reply:
Indeed that's much better thanks 🙏🏻 
(Skills issue cause I initially had copilot suggesting that and it did not worked so I went with the ugly regex)